frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.css

```diff
@@ -1,11 +1,11 @@
 
 .usa-form .ttahub-next-steps-repeater .usa-textarea {
-  /* Keep within bounds of navigator form with or without delete icon */
   max-width: 460px;
 }
 
 /* Outline a blank step onBlur or form Validation */
-.ttahub-next-steps-repeater .usa-form-group--error .blank-next-step textarea,
-.usa-form-group--error .ttahub-next-steps-repeater .blank-next-step textarea {
-    border: 3px solid #b50909;
-  }
\ No newline at end of file
+.ttahub-next-steps-repeater .usa-form-group--error .blank-next-step textarea ,
+.usa-form-group--error .ttahub-next-steps-repeater .blank-next-step textarea
+{
+    border: 2px solid #b50909;
```

Comment:
Can you convert this to a .scss file and use the brand color explicitly? Check out any other scss file for the syntax

Reply:
Updated to be a SCSS file and use the pre defined colors.